### PR TITLE
Add test for new large labels when in contour mode

### PR DESCRIPTION
**Pull request by jni**
# Description

I went through the trouble of creating a reproducible test case for [this imagesc post](https://forum.image.sc/t/data-specific-reason-for-indexerror-in-raw-to-displayed/60808) before realising that @Czaki had already fixed the issue in #3697 :joy: so I figured why waste a nice test case. :blush:

### CODE REVIEW

**Review by sofroniewn — APPROVED** [review]
LGTM!

**Review by sofroniewn — CHANGES_REQUESTED** [review]
Too fast! @jni can you move this test inside `napari/_tests` if you need to use `make_napari_viewer` or can you just make a labels layer if you want it to live here. Our headless tests are failing because of this one

**Comment by jni** [comment]
Iiinteresting! I saw the failure but didn't know how to interpret it. We should probably add this info (and a bunch of other stuff) to https://napari.org/developers/testing.html

Anyway, will update, thanks for the review! 😊

**Comment by tlambert03** [comment]
> Too fast! @jni can you move this test inside napari/_tests if you need to use make_napari_viewer or can you just make a labels layer if you want it to live here.

even better, just test `ViewerModel` if you don't need the Qt part of it

File: napari/layers/labels/_tests/test_labels.py [hunk 1]
```diff
@@ -563,6 +563,24 @@ def test_contour(input_data, expected_data_view):
     )
 
 
+def test_contour_large_new_labels(make_napari_viewer):
+    """Check that new labels larger than the lookup table work in contour mode.
+
+    References
+    ----------
+    [1]: https://forum.image.sc/t/data-specific-reason-for-indexerror-in-raw-to-displayed/60808
+    [2]: https://github.com/napari/napari/pull/3697
```
Comment (goanpeca) [inline napari/layers/labels/_tests/test_labels.py:573]: So lovely to put nicely formatted references @jni 😍 

We need more of these!

File: napari/layers/labels/_tests/test_labels.py [hunk 2]
```diff
@@ -563,6 +563,24 @@ def test_contour(input_data, expected_data_view):
...
+    """
+    viewer = make_napari_viewer()
+
+    labels = np.zeros((5, 10, 10), dtype=int)
+    labels[0, 4:6, 4:6] = 1
+    labels[4, 4:6, 4:6] = 1000
+    labels_layer = viewer.add_labels(labels)
+    labels_layer.contour = 1
+    viewer.dims.set_point(axis=0, value=4)
```
Comment (goanpeca) [inline napari/layers/labels/_tests/test_labels.py:583]: Do we need to assert something, or just that no error is raised?

In that case a comment could help :)

**Review by goanpeca — COMMENTED** [review]
🚀

**Comment by jni** [comment]
@sofroniewn I think my latest update will fix things... 🤞

**Review by sofroniewn — APPROVED** [review]
LGTM! All tests passing now, thanks!!

**Pull request merged**